src/correction/pfm.cpp

```diff
@@ -20,7 +20,7 @@
 namespace sgct::correction {
 
 Buffer generatePerEyeMeshFromPFMImage(const std::filesystem::path& path, const vec2& pos,
-                                      const vec2& size)
+                                      const vec2& size, bool textureRenderMode, int nEyes)
```

Comment:
Would it make more sense to have `bool isStereo` instead of `nEyes`?  It looks to me that `nEyes` should only have the values 1 or 2.

Reply:
Agreed, but then I decided to remove this entirely and just use the `textureRenderMode` bool to determine this state, because if true that's the only time the function isn't in stereo mode.